allennlp/modules/seq2seq_encoders/stacked_self_attention.py

```diff
@@ -83,14 +84,18 @@ def __init__(self,
             self.add_module(f"feedforward_{i}", feedfoward)
             self._feedfoward_layers.append(feedfoward)
 
+            layer_norm = LayerNorm(feedfoward.get_input_dim())
```

Comment:
Duplicating the variable name here makes it really easy to add bugs later - can you use a different name for this than you use below?